[debug] [2018-03-09T17:26:07.959Z] ----------------------------------------------------------------------
[debug] [2018-03-09T17:26:07.962Z] Command:       C:\Program Files\nodejs\node.exe C:\Users\fabiano.santana\AppData\Roaming\npm\node_modules\firebase-tools\bin\firebase login --reauth
[debug] [2018-03-09T17:26:07.962Z] CLI Version:   3.17.5
[debug] [2018-03-09T17:26:07.962Z] Platform:      win32
[debug] [2018-03-09T17:26:07.962Z] Node Version:  v9.4.0
[debug] [2018-03-09T17:26:07.962Z] Time:          Fri Mar 09 2018 14:26:07 GMT-0300 (Hora oficial do Brasil)
[debug] [2018-03-09T17:26:07.963Z] ----------------------------------------------------------------------
[debug] 
[info] 
[info] Visit this URL on any device to log in:
[info] https://accounts.google.com/o/oauth2/auth?client_id=563584335869-fgrhgmd47bqnekij5i8b5pr03ho849e6.apps.googleusercontent.com&scope=email%20openid%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcloudplatformprojects.readonly%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Ffirebase%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcloud-platform&response_type=code&state=762403259&redirect_uri=http%3A%2F%2Flocalhost%3A9005
[info] 
[info] Waiting for authentication...
[debug] [2018-03-09T17:26:23.942Z] >>> HTTP REQUEST POST https://accounts.google.com/o/oauth2/token  
 { code: '4/AAD7BePwJXSNHTsZhx7TrG6EW-97Uf9MlVjFilZGrWL4PRxTK5QpKy83Ptw5W6LXRSuPiwyh6DGuyPF_2la3j0I',
  client_id: '563584335869-fgrhgmd47bqnekij5i8b5pr03ho849e6.apps.googleusercontent.com',
  client_secret: 'j9iVZfS8kkCEFUPaAeJV0sAi',
  redirect_uri: 'http://localhost:9005',
  grant_type: 'authorization_code' } 
 Fri Mar 09 2018 14:26:23 GMT-0300 (Hora oficial do Brasil)
[debug] [2018-03-09T17:26:44.979Z] Token Fetch Error: Error
    at new FirebaseError (C:\Users\fabiano.santana\AppData\Roaming\npm\node_modules\firebase-tools\lib\error.js:11:17)
    at Request._callback (C:\Users\fabiano.santana\AppData\Roaming\npm\node_modules\firebase-tools\lib\api.js:34:23)
    at self.callback (C:\Users\fabiano.santana\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\request\request.js:186:22)
    at Request.emit (events.js:160:13)
    at Request.onRequestError (C:\Users\fabiano.santana\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\request\request.js:878:8)
    at ClientRequest.emit (events.js:160:13)
    at TLSSocket.socketErrorListener (_http_client.js:389:9)
    at TLSSocket.emit (events.js:160:13)
    at emitErrorNT (internal/streams/destroy.js:64:8)
    at process._tickCallback (internal/process/next_tick.js:152:19)
